python/tank/commands/core_upgrade.py

```diff
@@ -340,8 +359,15 @@ def get_update_status(self):
             # running updated version already
             return self.UP_TO_DATE
         else:
+            if (
```

Comment:
Wouldn't it be cleaner if we had
```
elif(...):
    # The config is immutable so we should not try updating it.
    return TankCoreUpdater.UPDATE_BLOCKED_BY_CONFIG
else:
   old code with the FIXME
```

What do you think?

Reply:
Yeah makes sense, I've made the change.